src/main/java/pages/FeedPage.java

```diff
@@ -0,0 +1,40 @@
+package pages;
+
+import com.codeborne.selenide.Condition;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+
+import java.time.Duration;
+
+import static com.codeborne.selenide.Selenide.$;
+
+public class FeedPage {
+    public final static String URL_FEED_PAGE = "https://moodpanda.com/Feed/";
+    public final static String UPDATE_MOOD_BUTTON_CSS = "#LinkUpdate";
+    public final static String GET_RATING_HAPPINESS = "//*[contains(@class,'p7670318')and contains(@class,'m1')]//span[contains(@class,'badge')]";
```

Comment:
don't use name of classes like 'p7670318' because more lileky they're auto generating (changing from time to time). Now tests are failed when searching for this locator